server/src/test/java/org/elasticsearch/index/mapper/GeoShapeFieldTypeTests.java

```diff
@@ -44,4 +45,13 @@ public void testFetchSourceValue() throws IOException {
         assertEquals(List.of(jsonLineString, jsonPoint), fetchSourceValue(mapper, sourceValue, null));
         assertEquals(List.of(wktLineString, wktPoint), fetchSourceValue(mapper, sourceValue, "wkt"));
     }
+
+    public void testFetchMalformedValue() throws IOException {
```

Comment:
Now that we don't automatically ignore values, it's become important to test malformed values at the field type level. Would you be up for adding similar tests to the other cartesian field types like (geopoint, legacy gepoint, point, and shape?)

Reply:
I've added these extra tests, which exposed an inconsistency in how point mappers deal with malformed values - have opened https://github.com/elastic/elasticsearch/issues/69601 to follow up.